### PR TITLE
FEAT: Add new parser RearLocalConf

### PR DESCRIPTION
**Pull request by TZ3070**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Add new parser ReaRLocalConf

### CODE REVIEW

File: insights/parsers/rear_conf.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    Examples:
+        >>> type(local_conf)
+        <class 'insights.parsers.rear_conf.ReaRLocalConf'>
+        >>> local_conf.data[0] == 'BACKUP_RESTORE_MOVE_AWAY_FILES=( /boot/grub/grubenv /boot/grub2/grubenv )'
+        True
+    """
+
+    def parse_content(self, content):
+        _content = get_active_lines(content)
```
Comment (xiangce) [inline insights/parsers/rear_conf.py:31]: `self.data` is not recommended to be used as it's too common to be an attr name.

Comment (xiangce) [reply insights/parsers/rear_conf.py:31]: as `TextFileOutput` is used as the base, you can just `pass` the parser unless you have new requires other than the base for this spec.

File: insights/parsers/rear_conf.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        >>> type(local_conf)
+        <class 'insights.parsers.rear_conf.RearLocalConf'>
+        >>> local_conf.lines[0] == 'BACKUP_RESTORE_MOVE_AWAY_FILES=( /boot/grub/grubenv /boot/grub2/grubenv )'
+        True
+    """
+
+    def parse_content(self, content):
+        if not content:
+            raise SkipComponent
+        super(RearLocalConf, self).parse_content(get_active_lines(content))
```
Comment (xiangce) [inline insights/parsers/rear_conf.py:33]:
```python
def parse_content(self, content):
   content = get_active_lines(content)
   if not content:
       raise Skip
   super(RearLocalConf, self).parse_content(content)```

**Review by JoySnow — APPROVED** [review]
LGTM

**Pull request merged**